### ISSUE TITLE
Adding vinculin antibody

### ISSUE THREAD

**Issue by Fabien-Maury** [obophenotype/human-phenotype-ontology#10399]
**Preferred term label:**
vinculin antibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against human vinculin.
(PMID:33411141|PMID:35876914)

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:90291|ORPHA:182090|ORPHA:801

**Your nano-attribution (ORCID)**
0000-0003-2854-0167

**Comment by pnrobinson** [comment]
@Fabien-Maury 
Adding new term: 
- ID: HP:6000645
- label: Anti-vinculin antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:33411141|PMID:35876914
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against vinculin.
- comment: 
- orcid: orcid:0000-0003-2854-0167
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10399
- synonyms:
